packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -127,6 +127,10 @@ export const getOptionStyles = memoizeFunction(
         wordWrap: 'break-word',
         overflowWrap: 'break-word',
         margin: '1px'
+      },
+      optionTextWrapper: {
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
       }

```

Comment:
Do we want to set this styling by default? Won't that potentially change default render when the text is longer than the width of the menu?